staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -200,25 +200,11 @@ func (o *RolloutStatusOptions) Run() error {
 		},
 	}
 
-	preconditionFunc := func(store cache.Store) (bool, error) {
```

Comment:
Thanks for working on getting this fixed @wojtek-t! Looks like your error is in this file:
```
vendor/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go:26:2: "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/api/errors" imported but not used as apierrors
exit status 1
```